assignments/prototypes.js

```diff
@@ -32,16 +58,66 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+
+function Humanoid(gameObj) {
+  characterStats.call(this, gameObj);
+  this.faction = gameObj.faction;
+  this.weapons  = gameObj.weapons;
+  this.language = gameObj.language;
+};
+
+Humanoid.prototype.greet = function() {
+  return `${this.name} offers a greeting in ${this.language}.`
+};
+
 /*
   * Inheritance chain: GameObject -> CharacterStats -> Humanoid
   * Instances of Humanoid should have all of the same properties as CharacterStats and GameObject.
   * Instances of CharacterStats should have all of the same properties as GameObject.
 */
 
+
+// Hero
+Hero.prototype.slice = function(villian) {
+  if (villian.hp > 0)
+  // return `${hero.name} hp -42`;
+  console.log("Optimus sliced Megatron! Megatron health dropped to:")
+  return villian.hp - 30; 
```

Comment:
To get this to work, try it this way:
  console.log("Optimus sliced Megatron! Megatron health dropped to:")
   villian.hp -= 30; 
  return villian.hp